rubocop-airbnb/.gitignore

```diff
@@ -0,0 +1,40 @@
+# Ignore bundler config
+/.bundle
+bundle
+vendor/bundle/
+vendor/rubocop/
+
+# Ignore the default SQLite database.
+/db/*.sqlite3
+
+# Ignore all logfiles and tempfiles.
+/log/*.log
+/tmp
+
+# Ignore development bundled assets
+.sass-cache/
+public/assets/
+
+/config/database.yml
+/config/configatron/secure.yml
+/config/initializers/aws_credentials.rb
+
+.DS_Store
+.rvmrc
+.ruby-version
+.*.swp
+.\#*
+\#*#
+node_modules
+npm-debug.log
+
+rubocop-airbnb*.gem
+
+# IDEs
+.idea
```

Comment:
It's not the end of the world if they are in here I recommend people put environment-specific ignore files such as editor files in their global .gitignore, not in each repo. The same advice applies to some of the other lines in this file, such as `#*#` and `.\#*` and `.*.swp`.

Reply:
Oh also this file should be moved up to the root

Reply:
Cleaning up the .gitignore before merging.